deploy/charts/README.md

```diff
@@ -0,0 +1,30 @@
+# Helm charts
+
+Each folder of this directory contains a single [helm chart](https://helm.sh/docs/topics/charts/).
+
+A single helm chart organizes a collection of templatized k8s yaml files.
+
+## Installing a Helm chart
+
+Installing a Helm chart means filling in the templates with a set of values and 
+applying the resources to a live cluster.
+
+Note: Helm uses k8s config for authentication. You can visit the GCP UI and click "CONNECT" on your cluster's page to get the command to configure the k8s config. Check if the k8s config points to the right cluster with `kubectl config current-context`.
+
+### Example 1: Install/update Mixer dev instance using local mixer
+
+Run the following after changes are made locally and are committed.
+```sh
+./scripts/push_binary.sh
```

Comment:
This is interesting... Forgive my ignorance, but I would have thought that you would need to run helm before push_binary.sh so the updated yaml files are deployed?

Reply:
push_binary is for creating a new mixer image based on local change.

The helm yaml files point to a specific image (by tag). The yamls assume that the specific image already exists in gcr. So the order is 1) push the image to gcr 2) call helm which refers to that image.

Added a comment in the readme!